prow/config/config_test.go

```diff
@@ -82,9 +82,10 @@ func TestContextMatches(t *testing.T) {
 
 func CheckRetest(t *testing.T, repo string, presubmits []Presubmit) {
 	for _, p := range presubmits {
-		expected := fmt.Sprintf("@k8s-bot %s test this", p.Name)
-		if p.RerunCommand != expected {
-			t.Errorf("%s in %s rerun_command: %s != expected: %s", repo, p.Name, p.RerunCommand, expected)
+		newCmd := fmt.Sprintf("/test %s", p.Name)
+		oldCmd := fmt.Sprintf("@k8s-bot %s test this", p.Name)
+		if p.RerunCommand != newCmd && p.RerunCommand != oldCmd {
```

Comment:
You are right - the `RerunCommand` does not need to handle `@k8s-bot test this`. This is because we only use `/test` for re-running the tests.